ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -34,7 +34,20 @@ public NotificationModelBase(INotificationService service)
         public Dictionary<string, NotifyError> NotifyErrorDictionary { get; set; }
 
 
-        public async Task<IActionResult> OnPostSubmitAsync()
+        public async Task<IActionResult> OnPostAsync(string actionName)
```

Comment:
Can you mention why we've made a decision to use this (as opposed to separate actions, which one could see as more idiomatic)?